br/pkg/backup/client.go

```diff
@@ -356,6 +356,17 @@ func BuildBackupRangeAndSchema(
 	return ranges, backupSchemas, nil
 }
 
+func skipUnsupportedDDLJob(job *model.Job) bool {
+	switch job.Type {
+	// Because TiDB V5.3.0 supports TableAttributes and TablePartitionAttributes.
+	// Incremental Backup need skip these DDLs to be compatible with previous br versions.
```

Comment:
change the comment to mention we're skipping it because placement is out of scope of backup, not just due to compatibility.

speaking of which, should these be filtered:

```go
	ActionCreatePlacementPolicy         ActionType = 51
	ActionAlterPlacementPolicy          ActionType = 52
	ActionDropPlacementPolicy           ActionType = 53
	ActionAlterTablePartitionPolicy     ActionType = 54
	ActionModifySchemaDefaultPlacement  ActionType = 55
	ActionAlterTablePlacement           ActionType = 56
```

Reply:
Add other DLL placementPolicy into skipUnsupportedDDLJob(), and add a testcase